src/Aquifer.Jobs/Services/SendGridEmailService.cs

```diff
@@ -101,7 +125,7 @@ public override string ToString()
         return $"""
                 {Message}
 
-                Email:
```

Comment:
This looks like it may be unintended?

Reply:
Yes.  Too much renaming.  Fixed: https://github.com/BiblioNexusStudio/aquifer-server/pull/529/commits/acd8a9d498c6109824ea2e52a901352630e21e9c